codespell_lib/_codespell.py

```diff
@@ -1053,7 +1057,9 @@ def main(*args: str) -> int:
     uri_ignore_words = parse_ignore_words_option(options.uri_ignore_words_list)
 
     if options.dictionary:
-        dictionaries = options.dictionary
+        dictionaries = []
+        for d in options.dictionary:
+            dictionaries.extend(d.split(","))
     else:
         dictionaries = ["-"]
     use_dictionaries = []
```

Comment:
Remove the `if options.dictionary:` line above and then:
```suggestion
    dictionaries = []
    for d in options.dictionary or []:
        dictionaries.extend(d.split(","))
    dictionaries = dictionaries or ["-"]
    use_dictionaries = []
```